.github/ISSUE_TEMPLATE/issue.md

```diff
@@ -0,0 +1,24 @@
+---
+name: Tracking issue
+about: Use this template for tracking complaints.
+title: "[DATE]: [ISSUE NAME]"
+labels: tracking issue, needs triage
+assignees: wwelling
+---
+
+1. What is the type of this issue?
+    - [ ] Client
+
+        - [ ] Documentation: documentation is missing, incomplete, incorrect, ambiguous, or incomprehensible.
+        - [ ] Functional: no longer able to satisfy use case due to newly incurred limitations.
+        - [ ] Behavioral: something happened not expected.
```

Comment:
```suggestion
        - [ ] Behavioral: something happened that is not expected.
```
The grammer should be improved here.

Adding "that is" should fix the grammar.

Or consider "Something unexpected happened."?
Or consider "Something unexpected or unintended happened."?